src/components/HelpCenterBar/InfoDialog.tsx

```diff
@@ -34,36 +39,65 @@ type InfoDialogProps = {
 
 export function InfoDialog({ open, onClose }: InfoDialogProps) {
   const { t } = useTranslation();
+  const [showRevision, setShowRevision] = useState(false);
 
   const version = getEnvironment('REACT_APP_VERSION', 'unset');
   const revision = getEnvironment('REACT_APP_REVISION', 'unset');
 
+  const onClickShowRevision = useCallback(() => {
+    setShowRevision(!showRevision);
+  }, [showRevision]);
+
   const dialogTitleId = useId();
   const dialogDescriptionId = useId();
 
   return (
     <Dialog
       open={open}
       onClose={onClose}
+      fullWidth
+      maxWidth="sm"
       aria-labelledby={dialogTitleId}
       aria-describedby={dialogDescriptionId}
     >
-      <DialogTitle id={dialogTitleId}>
-        {t('helpCenter.info.title', 'About NeoBoard')}
-      </DialogTitle>
-      <DialogContent>
+      <Stack alignItems="baseline" direction="row">
+        <DialogTitle component="h3" id={dialogTitleId} sx={{ flex: 1 }}>
+          {t('helpCenter.info.title', 'About NeoBoard')}
+        </DialogTitle>
+        <Tooltip title={t('helpCenter.info.close', 'Close')}>
+          <IconButton
+            sx={{ mr: 3 }}
+            aria-label={t('helpCenter.info.close', 'Close')}
+            onClick={onClose}
+          >
+            <CloseIcon />
+          </IconButton>
+        </Tooltip>
+      </Stack>
+      <DialogContent sx={{ pt: 0 }}>
         <DialogContentText
           id={dialogDescriptionId}
           sx={{ wordBreak: 'break-all' }}
         >
-          {t('helpCenter.info.version', 'Version {{version}} ({{revision}})', {
+          {t('helpCenter.info.version', 'Version {{version}}', {
             version,
             revision,
           })}
         </DialogContentText>
+        <Button sx={{ pl: 0 }} variant="text" onClick={onClickShowRevision}>
+          {!showRevision
+            ? t('helpCenter.info.showMore', 'Show more')
+            : t('helpCenter.info.showLess', 'Show less')}
+        </Button>
```

Comment:
The button looks odd with the padding that is removed only on the left side. How about using a link instead if a button has the wrong layout?

```tsx
<Typography my={1} fontWeight="bold">
  <Link
    component="button"
    onClick={onClickShowRevision}
    underline="none"
  >
    {!showRevision
      ? t('helpCenter.info.showMore', 'Show more')
      : t('helpCenter.info.showLess', 'Show less')}
  </Link>
</Typography>
```